app/services/trainees/create_from_dttp.rb

```diff
@@ -135,8 +138,19 @@ def undergrad_level?
       dttp_trainee.latest_placement_assignment.response["dfe_courselevel"] == Dttp::Params::PlacementAssignment::COURSE_LEVEL_UG
     end
 
+    def trn_present?
+      dttp_trainee.trn.present?
+    end
+
     def trn
-      dttp_trainee.trn == INVALID_TRN ? nil : dttp_trainee.trn
+      if trn_present?
+
+        dttp_trainee.trn.strip!
```

Comment:
using `strip!` might modify the original trn value in the database against `response->'dfe_trn'` if whitespace exists.
would it be possible to try this with the non-bang version please?